js/app_contact_form_handler.js

```diff
@@ -0,0 +1,78 @@
+// get all data in form and return object
+function getContactFormData() {
```

Comment:
we should not be copy-pasting the form handler code when we can "include" it directly from the CDN using https://rawgit.com to keep things "DRY".

e.g: https://cdn.rawgit.com/dwyl/learn-to-send-email-via-google-script-html-no-server/faa599f47774ca6397ece7d57a08a247f0469186/form-submission-handler.js
For a _specific_ version of the code.

Or without the Git Hash (_if we always want to use the latest version of the code_):
https://cdn.rawgit.com/dwyl/learn-to-send-email-via-google-script-html-no-server/master/form-submission-handler.js